api/api-doc/raft_group0.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Better to implement https://github.com/scylladb/scylladb/issues/12008 instead adding new rest apis.

Reply:
#12008 is not a trivial project - the proposed table would contain all Raft groups and more information than just configuration. I'm also not familiar with the virtual table API that well. It would take me some time, which we don't have before the 5.2 release, to do it. The REST API doesn't hurt and it solves the current problem. I think we should avoid scope creep and keep this PR focused.

Reply:
It may start as one group (that all we have anyways for now). I do not know much about virtual tables either, but we have a bunch so the infra should be in place to create more.